zookeeper-client/zookeeper-client-c/include/proto.h

```diff
@@ -40,6 +40,8 @@ extern "C" {
 #define ZOO_RECONFIG_OP 16
 #define ZOO_CHECK_WATCHES 17
 #define ZOO_REMOVE_WATCHES 18
+#define ZOO_CREATE_CONTAINER_OP 19
+#define ZOO_DELETE_CONTAINER_OP 20
```

Comment:
You might not want to add the opcode for delete, because that is not a command which client is supposed to submit. You've only added support for creation anyway.